ci/test_self_hosted_service.sh

```diff
@@ -19,6 +19,9 @@ set -euo pipefail
 
 source rapids-init-pip
 
+# Install Boost and TBB
+bash ci/utils/install_boost_tbb.sh
```

Comment:
Lets comment and see whether boost is getting coupled with wheel or needs explicit installation.